infrastructure/storage-provisioning/README.md

```diff
@@ -85,3 +85,32 @@ After that you can delete test with commands
 $ kubectl delete -f examples/wordpress.yaml
 $ kubectl delete -f examples/mysql.yaml
 ```
+
+## Rook NFS
+The personal storage disk attached to VMs and containers is based on the NFS server provided by Rook-Ceph ([here](https://rook.io/docs/rook/v1.10/Storage-Configuration/NFS/nfs/)), which can be configured with the `CephNFS` CRD of the NFS-Ganesha server.
+
+The user's storage is dynamically provisioned through a `StorageClass` that contains a set of parameters that tell the Rook Operator to create also a NFS share; this is later mounted from user's VMs and containers though the NFS protocol.
+
+To create the necessary resources:
+```bash
+$ kubectl create -f nfs-manifests/nfs.yaml
+$ kubectl create -f nfs-manifests/storageclass.yaml
+```
+
+Be aware that to use this feature the system must be configured with:
+- Rook v1.9.1 or greather
+- Ceph v16.2.7 or greather
+
+### Using the NFS service
+Mounting NFS shares inside Virtual Machines may require an additional software package, e.g., `nfs-common` package for Debian-based VMs, and `nfs-utils` for CentOS-based VMs.
+
+The DNS resolution for the service name on Container instances is done by the software running on the physical server (i.e., kubelet), which does not have access to the Kubernetes cluster's DNS server.
+To allow the host machine to resolve the DNS name for the NFS server you need to add the proper record to the host's `/etc/resolv.conf`, either manually or with an automated process (e.g., ansible).
```

Comment:
```suggestion
To allow the host machine to resolve such DNS name, several solution can be implemented, for example adding the proper record to the host's `/etc/resolv.conf`, either manually or with an automated process (e.g., ansible). This operation should to be done right after the creation or modification of the NFS service, that would cause the allocation of the service cluster IP address.
```